### DISCUSSION TITLE
TypeError: undefined is not an object (evaluating 'document.body.addEventListener')

### DISCUSSION THREAD

**Discussion by ghost** [GeekyAnts/NativeBase#4841] [Q&A]
I am currently using NativeBase v2 (mostly used ActionSheet, Section, Content) in my project and trying to migrate to v3.
Platform - iOS
1. Removed all the references native-base v2
2. Removed v2 from package.json
3. Clean and Build project works fine
4. Added native-base@3.2.1 to package.json
5. Clean and Build project

ERROR  TypeError: undefined is not an object (evaluating 'document.body.addEventListener')
ERROR  Invariant Violation: Module AppRegistry is not a registered callable module (calling runApplication). A frequent cause of the error is that the application entry file path is incorrect. 
      This can also happen when the JS bundle is corrupt or there is an early initialization error when loading React Native.
ERROR  Invariant Violation: Module AppRegistry is not a registered callable module (calling runApplication). A frequent cause of the error is that the application entry file path is incorrect. 
      This can also happen when the JS bundle is corrupt or there is an early initialization error when loading React Native.

Any Help will be greatly appreciated.

**Comment by sanna231** [comment]
Hey @konampruthvirajkumar , It would be great if you can provide snack / codesandbox link ? Also [this](https://docs.nativebase.io/next/migration/v3) migration guide might help you upgrading to v3.
